net/redisclient.go

```diff
@@ -197,6 +203,31 @@ func NewRendezvousVnodes(shards []string) redis.ConsistentHash {
 	return rendezvousVnodes{rendezvous.New(vshards, xxhash.Sum64String), table}
 }
 
+func (r *RedisRingClient) updateIPsFromURL() func() []string {
```

Comment:
Why this is part of `net` package?

Reply:
see also https://github.com/zalando/skipper/blob/437633129a60549e16d8cf93098b6301d06f5e28/skipper.go#L1279-L1291 , similar would be to add a func in skipper.go.